apiary.apib

```diff
@@ -0,0 +1,193 @@
+FORMAT: 1A
```

Comment:
Shouldn't we document the (known) 5xx cases here, too?

Reply:
Done